bindings/C/AUD_Sound.h

```diff
@@ -420,6 +420,39 @@ extern AUD_API AUD_Sound* AUD_Sound_mutable(AUD_Sound* sound);
      * \return A handle of the time-stretched, pitch scaled sound.
      */
     extern AUD_API AUD_Sound* AUD_Sound_timeStretchPitchScale(AUD_Sound* sound, double timeRatio, double pitchScale, AUD_StretcherQuality quality, bool preserveFormant);
+
+    /**
+     * Time-stretches and pitch scales a sound with animation support
+     * \param sound The handle of the sound.
+     * \param timeRatio The factor by which to stretch or compress time.
+     * \param pitchScale The factor by which to adjust the pitch.
+     * \param quality The processing quality level of the stretcher.
+     * \param preserveFormant Whether to preserve the vocal formants for the stretcher.
+     * \return A handle of the time-stretched, pitch scaled sound.
+     */
+    extern AUD_API AUD_Sound* AUD_Sound_animateableTimeStretchPitchScale(AUD_Sound* sound, double timeRatio, double pitchScale, AUD_StretcherQuality quality,
```

Comment:
Since this is a C API and C doesn't have booleans by default, could you please change the type to char like in the other functions? Same for `AUD_Sound_timeStretchPitchScale` where I missed it in the last review please.

Reply:
You missed just this line here.